transport-native-epoll/src/main/java/io/netty/channel/unix/FileDescriptor.java

```diff
@@ -80,4 +83,14 @@ public int hashCode() {
     }
 
     private static native int close(int fd);
+
+    public static FileDescriptor from(File file) throws IOException {
```

Comment:
- Could we also add `public static FileDescriptor from(String path) throws IOException` which is a simple shortcut to this method?
- Please do a null-check on `file`.